### PR TITLE
retroshare: Update to 0.6.0

### PR DESCRIPTION
**Pull request by abelboldu**

### CODE REVIEW

**Comment by Vaelatern** [comment]
Please squash into one commit.

File: srcpkgs/retroshare/template [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+revision=1
 homepage="http://retroshare.sourceforge.net/"
 short_desc="A Decentralized Social Sharing Network"
 maintainer="Abel Vdo <vdo@greyfaze.net>"
 license="GPL-3"
 hostmakedepends="pkg-config qt-qmake"
-makedepends="qt-devel libressl-devel libssh-devel zlib-devel bzip2-devel sqlite-devel libupnp-devel libgnome-keyring-devel libmicrohttpd-devel sqlcipher-devel libXScrnSaver-devel libxslt-devel libcurl-devel"
-distfiles="${SOURCEFORGE_SITE}/retroshare/RetroShare/0.6.0-rc2/${pkgname}_${version}~8551_src.tgz"
-checksum="2320676da905de6c48b01eda611811965277ffa1d5ddbb387aa8f0414c2de050"
+makedepends="qt-devel libressl-devel libssh-devel zlib-devel bzip2-devel sqlite-devel libupnp-devel libgnome-keyring-devel libmicrohttpd-devel sqlcipher-devel libXScrnSaver-devel libxslt-devel libcurl-devel libavcodec"
```
Comment (xtraeme) [inline srcpkgs/retroshare/template]: `libavcodec` is not a real dependency that can't be used without its headers, use `ffmpeg-devel` instead.

**Comment by Vaelatern** [comment]
Please fix the build errors. Probably a matter of talking to upstream.

**Comment by abelboldu** [comment]
Reported:
https://github.com/RetroShare/RetroShare/issues/349

**Comment by xtraeme** [comment]
retroshare was updated in 05f89b7f0bae5598e345f52ea1c07e393d6d6a62

Thanks anyway

**Pull request closed without merging**